### ISSUE TITLE
[Doc] Update content for pre-defined roles

### ISSUE THREAD

**Issue by Minoli** [wso2/product-apim#2884]
**Description:**

***Not fulfilling [checklist items](https://docs.google.com/spreadsheets/d/1l6YKXSbmtykvvn_NvX6uJbXSsZvpT8jn72Qoi_FoJq8/edit#gid=1221574205):***
Is the content up to date and accurate?

***Related task:***
Create roles

***Issue:***
The creator and publisher roles are available by default. Docs mention creating same new roles with permissions. 

***Proposed Solution:***
State that creator and publisher roles are added by default with the listed permissions.
	
***Doc Link:***
https://docs.wso2.com/display/AM210/Quick+Start+Guide#QuickStartGuide-DeepdivingintotheAPIManager

**Suggested Labels:**
Bug, Docs, 2.1.0

**Affected Product Version:**
2.1.0

**Comment by shaniR** [comment]
Yes creator and publisher roles are added by default. They are internal roles created in the internal carbon database. Internal roles cannot be mapped with external user stores. In order to do this, you will have to create a role on the User store. The document describes how to create a role from scratch.
